### PR TITLE
Implement Exercise: Armstrong-numbers

### PR DESCRIPTION
**Pull request by Dwivedirahul44**
ref:https://github.com/exercism/abap/issues/223

### CODE REVIEW

**Comment by larshp** [comment]
`abap.builtin.ipow is not a function` missing in transpiler, I will fix

**Comment by Dwivedirahul44** [comment]
> `abap.builtin.ipow is not a function` missing in transpiler, I will fix
> 
> tho, floating point might be a problem, hmm

I can add workaround using a**b

File: exercises/practice/armstrong-numbers/zcl_armstrong_numbers.clas.testclasses.abap [hunk 1]
```diff
@@ -0,0 +1,78 @@
+*"* use this source file for your ABAP unit test classes
+*"* use this source file for your ABAP unit test classes
```
Comment (larshp) [inline exercises/practice/armstrong-numbers/zcl_armstrong_numbers.clas.testclasses.abap:2]:
```suggestion
```

**Comment by larshp** [comment]
image

File: exercises/practice/armstrong-numbers/zcl_armstrong_numbers.clas.testclasses.abap (hunk 1 above)
Comment (Dwivedirahul44) [reply exercises/practice/armstrong-numbers/zcl_armstrong_numbers.clas.testclasses.abap]: changed,thanks for pointing out

**Comment by Dwivedirahul44** [comment]
> image

There was typo , fixed and tested.

**Review by larshp — APPROVED** [review]
👍

**Comment by larshp** [comment]
@Dwivedirahul44 take care of brancing, this PR should ideally not contain any commits regarding the acronym exercise

**Comment by Dwivedirahul44** [comment]
thank you!

**Pull request merged**